vep/README.md

```diff
@@ -1,77 +1,110 @@
+   receive an email once the transfer is completed.
 
-   [references]
-   vep_mount = "gs://cpg-common-main/references/vep/105.0/mount"
-   ```
+## Processing Local Cache
 
-   The VEP cache bundle is huge, so we use Hail Batch to copy it from Ensemble FTP servers and the Broad Institute servers to the CPG reference GCP bucket:
+1. Open the cache tarball 
 
-    ```bash
-    analysis-runner --dataset common --access-level full --description "Build resources for VEP $VEP_VERSION" --output-dir=vep/$VEP_VERSION --config=$HOME/tmp/vep_$VEP_VERSION.toml python3 copy-references.py $VEP_VERSION
-    ```
-
-1. If you want to permanently update the default VEP version for [`production-pipelines`](https://github.com/populationgenomics/production-pipelines) workflows:
-   * Update the version for `vep` in [`images.toml`](https://github.com/populationgenomics/images/blob/main/images.toml). This map translates to `cpg_utils.image_path('vep')` used in Hail Batch workflows.
-   * Update the version for `vep_mount` in [`references.py`](https://github.com/populationgenomics/references/blob/main/references.py). This map translates to `cpg_utils.reference_path('vep_mount')` used in Hail Batch workflows.
-
-## Running using Hail Batch
-
-Standard Hail Query `hl.vep` does not support Batch Backend, so we use a workaround for that:
-
-* Split input VCF into partitions using an interval list generated by Picard tools; 
-* Submit Hail Batch jobs that call VEP with reference data in a mounted volume `vep_mount`, as well as the previously built `vep` image;
-* For the VEP command, we parametrise it to write results using JSON format; 
-* Finally, we gather the JSON results into a Hail Table using a pre-prepared schema, which is borrowed from Hail Query's `hl.vep`. 
-
-The whole method is implemented in [CPG workflows](https://github.com/populationgenomics/production-pipelines/blob/main/cpg_workflows/jobs/vep.py) as part of the Seqr Loader workflow. See example of usage in [test/test-batch-backend.py](test/test-batch-backend.py).
-
-## Running using `hl.vep`
-
-The Query's `hl.vep` function works only with the Spark backend on a Dataproc cluster. To make it work with a custom VEP version, the following extra steps are required.
-
-1. The dataproc initialisation script in this repository [dataproc-init.sh](dataproc-init.sh) is copied from the [Hail codebase](https://github.com/hail-is/hail/blob/cc0a051740f4de08408e6a2094ffcb1c3158ee9c/hail/python/hailtop/hailctl/dataproc/resources/vep-GRCh38.sh) and adjusted to pull the image from CPG artefact registry, and parameterised by `VEP_VERSION`.
-
-Script is parametrised with `__VEP_VERSION__`, so pass it through `sed` when copying to the bucket:
-
-```shell
-cat dataproc-init.sh | sed "s/__VEP_VERSION__/${VEP_VERSION}/g" | gsutil cp - gs://cpg-common-main/references/vep/${VEP_VERSION}/dataproc/init.sh
+```commandline
+tar xzf homo_sapiens_vep_110_GRCh38.tar.gz
```

Comment:
How big does this file roughly become, should we recommend a user has at least `X GB` space available?

Reply:
Added a bit more detail here - it's ~20GB and unpacks to a second 20GB directory, with a little bit of mischief during the expansion.